Source/Core/DolphinQt/Config/Mapping/MappingIndicator.cpp

```diff
@@ -184,6 +190,22 @@ bool IsPointOutsideCalibration(Common::DVec2 point, ControllerEmu::ReshapableInp
   return current_radius > input_radius * ALLOWED_ERROR;
 }
 
+template <typename F>
+void GenerateFibonacciSphere(int point_count, F&& callback)
+{
+  const float golden_angle = MathUtil::PI * (3.f - std::sqrt(5.f));
```

Comment:
I don't suppose this could be made `constexpr` at some point, could it? I'm pretty sure `MathUtil::PI` isn't `constexpr` right now, but I don't see any reason why it couldn't be (plus it is still usable at runtime in non-`constexpr` context if necessary).

Reply:
`PI` is constexpr but `std::sqrt` isn't, unfortunately.

Reply:
Awww, too bad. Maybe in a later standard.